### PR TITLE
Feature/disabled prop type

### PR DESCRIPTION
**Pull request by brentoneill**
I have added a `disabled` prop type that you can apply to a `<RadioButton>`  component as a means to disable selection of that button. 

By adding a prop of `disabled={true}`, that `<RadioButton>` will no longer trigger the onChange event when clicked and will have some basic "disabled" styles added to it: `cursor: not-allowed` and a light grey `color` and `border-color` treatment.

Please see example.html for implementation and request any changes after review.

Thank you!

### CODE REVIEW

File: index.jsx [hunk 1]
```diff
@@ -141,7 +158,7 @@ class RadioIcon extends Component {
         borderWidth: 2,
         borderRadius: '50%',
         borderStyle: 'solid',
-        borderColor: rootColor || '#9E9E9E',
+        borderColor: disabled ? '#e1e1e1' : (rootColor || '#9E9E9E'),
```
Comment (mu29) [inline index.jsx]: How about adding `disabledColor` prop rather than just using `#e1e1e1`?

**Comment by mu29** [comment]
Awesome!
I'm sorry to bother you, but can you please add it in the README?

**Comment by brentoneill** [comment]
@mu29 - sure thing. 

I'll add the disabledColor prop type as well!

**Comment by mu29** [comment]
I will publish new version when I go to work :)

**Pull request merged**